tools/setup/emoji/emoji_names.py

```diff
@@ -868,26 +1361,54 @@
     "1f91c": {"canonical_name": "right_fist", "aliases": []},
     "1f91d": {"canonical_name": "handshake", "aliases": ["done_deal"]},
     "1f91e": {"canonical_name": "fingers_crossed", "aliases": []},
+    "1f91f": {"canonical_name": "love-you_gesture", "aliases": ["ily"]},
     "1f920": {"canonical_name": "cowboy", "aliases": []},
     "1f921": {"canonical_name": "clown", "aliases": []},
     "1f922": {"canonical_name": "nauseated", "aliases": ["queasy"]},
     "1f923": {"canonical_name": "rolling_on_the_floor_laughing", "aliases": ["rofl"]},
     "1f924": {"canonical_name": "drooling", "aliases": []},
     "1f925": {"canonical_name": "lying", "aliases": []},
+    "1f926-200d-2640": {"canonical_name": "woman_facepalming", "aliases": []},
+    "1f926-200d-2642": {"canonical_name": "man_facepalming", "aliases": []},
     "1f926": {"canonical_name": "face_palm", "aliases": []},
     "1f927": {"canonical_name": "sneezing", "aliases": []},
+    "1f928": {"canonical_name": "face_with_raised_eyebrow", "aliases": ["distrust", "skeptic"]},
+    "1f929": {"canonical_name": "star-struck", "aliases": []},
```

Comment:
I think we might treat `-`, ` `, and `_` as equivalent in our matching algorithm implementation -- should this be `star_struck`? I don't think we should intentionally have any `-`s in the `canonical_name` field.

Reply:
It's `star-struck` on Slack and Matrix (both using iamcal). It would be harder to bridge the reactions between the platforms if the canonical name is manually set to `star_struck`.

Reply:
Any code that assumes emoji are represented by names is already wrong, but any code that assumes emoji are represented by the *same* name on *three* wrong platforms is *quadruply* wrong. Colon strings are not a stable interchange format.

Reply:
The colon strings in Slack and Matrix are unlikely to change because they have been used in production for a while, and changing them would break compatibility. But I suppose my concern is easily solvable by using `zerver.lib.emoji.name_to_codepoint` in the bridge.

But then again, having the new colon strings that deviate from the most common standards would make it harder to bridge when adding emoji support in Matterbridge. To bridge reactions between Slack and Matrix, it doesn't require any mapping of name -> codepoint -> name.

Reply:
Emoji are Unicode characters. Literally the whole point of Unicode is to avoid these kind of stupid interchange problems. There is no good reason to map a codepoint that works everywhere to a name that’s only meaningful to some specific applications.

Reply:
It's not my choice that Slack and Mattermost (not sure about Matrix) have colon string reactions in their message JSON. As such, it is necessary to know what naming scheme they use anyway.

Reply:
Any code that importing/mirroring content between Zulip and third-party services will need to do something to map names for emoji, because every service uses their own set of names. It shouldn't be too hard to do name=>codepoint=>other name translations (or probably better in most cases, name => codepoint translations, i.e. just replacing `:star-struck:` in a slack message with the 🤩 unicode character) as needed in tools whose purpose is to translate between different protocols. 

However, we still need to answer my original question: Does `star-struck` and anything else with a `-` in it need to be spelled `star_struck` in order to not be broken/unusable in Zulip's emoji system?

Reply:
I think it was usable before, but I switched it to be all underscores for consistency.